.reduxrc

```diff
@@ -0,0 +1,8 @@
+{
+  "sourceBase":"src",
+  "testBase":"tests",
+  "smartPath":"containers",
+  "dumbPath":"components",
+  "fileCasing":"pascal"
```

Comment:
I choose to use pascal case for filenames by default because it seems like thats what the starter kit uses.  This means if someone does something like: `redux g dumb my-component` the file names will be: `src/components/MyComponent.js`.  If the `fileCasing` setting is set to default then it will use whatever the user enters into the generator for the filename.  I.E. in this example it would of generated `src/components/my-component.js`.  Can you confirm this is the appropriate functionality you want?